lib/app.js

```diff
@@ -1,46 +1,107 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const content = require('./content');
+const c = require('./content');
 const fetch = require('r2');
 
 module.exports = settings => {
   const app = express();
   app.use(bodyParser.json());
 
+  // I have the right now logged in user in meta
+  // this is the user making changes
+
+  const service = {
+    whitelist(json) {
+      let tos = [];
+
+      switch (json.meta.next) {
+        case 'autoresolved': {
+          tos.push({
+            name: `${json.meta.user.profile.firstName} ${
+              json.meta.user.profile.lastName
+            }`,
+            to: 'swagbag@swagbag.club', //json.meta.user.profile.email
+            template: 'notification_task_change'
+          });
+          break;
+        }
+        case 'with-ntco': {
+          // I need to send it to two people - applicant and ntco
+          console.log('SEND TO TWO PEOPLE');
+          // I still have the basci user logged in - I need to notify him as well
+          tos.push({
+            name: `${json.meta.user.profile.firstName} ${
+              json.meta.user.profile.lastName
+            }`,
+            to: 'swagbag@swagbag.club', //json.meta.user.profile.email
+            template: 'notification_task_change'
+          });
+          // how do I find the NTCO ?
```

Comment:
query the database. You'll be able to find out the establishmentId from the task then you'll need to query roles table where establishmentId === task.data.establishmentId && role.type === 'ntco'